src/python/deepgnn/pytorch/common/consts.py

```diff
@@ -10,9 +10,6 @@
 ENCODER_TYPES = "encoder_types"
 FANOUTS = "fanouts"
 FEATURE_ENCODER_STR = "feature_encoder"
-FP16_AMP = "amp"
-FP16_APEX = "apex"
```

Comment:
This is used for running twinbert models from deep speed I believe:
https://github.com/microsoft/DeepGNN/blob/8ec5b33dbe3d4311f4408ec1f8947149dc7d0ea7/src/python/deepgnn/pytorch/encoding/twinbert/encoder.py#L34

I don't see updates in these files though. Why we don't need to change them?

Reply:
FP16 is boolean in ray and it can/should be used in every model.

Reply:
My comment is about amp/apex mode, not fp16.

Reply:
Apex is fp16 for GPU, we dont run examples on GPU. In fact this example has not worked and still needs a dataset to make it work so can we just remove it?

Reply:
sure, feel free to remove it then.